### ISSUE TITLE
Update dependency to remove vulnerability.

### ISSUE THREAD

**Issue by ZandorSabino** [testdouble/scripty#112]
## Description
The async dependency has a vulnerability in the current version of the project, which has been resolved as of version 2.6.4.
```

Vulnerability: **🔴high** [Prototype Pollution in async.](https://www.npmjs.com/advisories/1070440)
Dependency: **async**

```
### Risk
Could allow a malicious user to gain privileges through the mapValues() method.

**Comment by rosston** [comment]
I think it's worth noting that the current version requirement for `async` (`^2.6.1`) is fully compatible with an end user running `async` 2.6.4 in their own project using `scripty`. But I'm also not against upping the requirement explicitly in the package.json, so I'll go ahead and move your PR forward. Thanks for working on this!

**Comment by rosston** [comment]
Released with 2.1.1! 🎉
